tests/Aggregation/TermsTest.php

```diff
@@ -162,4 +162,15 @@ private function getIndex(): Index
 
         return $index;
     }
+
+    /**
+     * @group functional
```

Comment:
This does not look like a functional test. As far as I can see, no call is made to Elasticsearch. Any chance you could expand the test in a way that a call to Elasticsearch is made to confirm the functionality.

Reply:
You are right. My mistake. Updated.